ext/RMagick/rmfill.c

```diff
@@ -130,14 +130,14 @@ point_fill(
     green_step = ((MagickRealType)stop_color->green - (MagickRealType)start_color->green) / steps;
     blue_step  = ((MagickRealType)stop_color->blue  - (MagickRealType)start_color->blue)  / steps;
 
-    for (y = 0; y < image->rows; y++)
+    for (y = 0; y < (ssize_t) image->rows; y++)
     {
         PixelPacket *row_pixels;
 
-        row_pixels = QueueAuthenticPixels(image, 0, (long int)y, image->columns, 1, exception);
+        row_pixels = QueueAuthenticPixels(image, 0, y, image->columns, 1, exception);
         CHECK_EXCEPTION()
 
-        for (x = 0; x < image->columns; x++)
+        for (x = 0; x < (ssize_t) image->columns; x++)
```

Comment:
unnecessary type cast.
```suggestion
        for (x = 0; x < image->columns; x++)
```